src/main/fc/fc_tasks.c

```diff
@@ -586,6 +592,15 @@ cfTask_t cfTasks[TASK_COUNT] = {
     },
 #endif
 
+#ifdef RUNCAM_SPLIT_SUPPORT
+    [TASK_RCSPLIT_SUPPORT] = {
+        .taskName = "RCSPLIT",
+        .taskFunc = rcsplitProcess,
+        .desiredPeriod = TASK_PERIOD_HZ(100),        // 100 Hz, 10ms
```

Comment:
RCSplit should use common BOX / rcMode implementation, without reimplementing it. Only problem is that all bits are currently used, so some extension will be necessary.